wbooks/src/app/index.tsx

```diff
@@ -10,38 +10,89 @@
 import 'react-native-gesture-handler';
 import React from 'react';
 import { createStackNavigator } from '@react-navigation/stack';
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 import { NavigationContainer } from '@react-navigation/native';
 import { RootStackParamList } from '@constants/types';
-import Colors from '@constants/colors';
 import { Image } from 'react-native';
+import Colors from '@constants/colors';
+import Library from '@screens/Library';
+import Settings from '@screens/Settings';
+import BookDetail from '@screens/BookDetail';
 
-import Library from './screens/Library';
-import BookDetail from './screens/BookDetail';
 import styles from './style';
 
-const Stack = createStackNavigator<RootStackParamList>();
-
 const HeaderBackground = () => (
   <Image source={require('@assets/General/bc_nav_bar.png')} resizeMode="contain" style={styles.header} />
 );
 
 const BackButton = () => <Image source={require('@assets/NavigationBar/ic_back.png')} />;
 
+const LibraryStack = createStackNavigator<RootStackParamList>();
+
+function LibraryStackScreen() {
+  return (
+    <LibraryStack.Navigator screenOptions={{ headerBackImage: () => <BackButton /> }}>
+      <LibraryStack.Screen name="Library" component={Library} />
+      <LibraryStack.Screen
+        name="BookDetail"
+        component={BookDetail}
+        options={{
+          title: 'BOOK DETAIL',
+          headerTintColor: Colors.white,
+          headerBackground: () => <HeaderBackground />
+        }}
+      />
+    </LibraryStack.Navigator>
+  );
+}
+
+const SettingsStack = createStackNavigator();
+
+function SettingsStackScreen() {
+  return (
+    <SettingsStack.Navigator>
+      <SettingsStack.Screen name="Settings" component={Settings} />
+    </SettingsStack.Navigator>
+  );
+}
+
+const Tab = createBottomTabNavigator();
+
 function App() {
   return (
     <NavigationContainer>
-      <Stack.Navigator screenOptions={{ headerBackImage: () => <BackButton /> }}>
-        <Stack.Screen name="Library" component={Library} />
-        <Stack.Screen
-          name="BookDetail"
-          component={BookDetail}
+      <Tab.Navigator
+        tabBarOptions={{
+          activeTintColor: Colors.primary,
+          inactiveTintColor: Colors.grey
+        }}>
+        <Tab.Screen
+          name="Library"
+          component={LibraryStackScreen}
+          options={{
+            tabBarIcon: ({ focused }) => {
```

Comment:
No, 2 things:

* Create a generic `TabBarIcon` component, you can use like this:
```
      screenOptions={({ route }) => ({
        tabBarIcon: ({ focused }) => <TabIcon focused={focused} route={route} />
      })}
```
* Don't repeat the same logic for all the tabs, always improve the code and avoid repeating